src/contracts/acceptProposal.tsx

```diff
@@ -95,14 +80,8 @@ export const validateProposal = async (
           throw new Error('Approve Transaction failed');
         }
       }
-      const { request } = await publicClient.simulateContract({
-        address: config.contracts.talentLayerEscrow,
-        abi: TalentLayerEscrow.abi,
-        functionName: 'createTransaction',
-        args: [parseInt(serviceId, 10), parseInt(proposalId, 10), metaEvidenceCid, cid],
-        account: walletClient.account?.address,
-      });
-      const tx2 = await walletClient.writeContract(request);
+
+      const { tx: tx2 } = await tlClient.escrow.approve(serviceId, proposalId, metaEvidenceCid);
```

Comment:
you miss a really important argument here, the cid, check the diff, this can't work like that